src/components/ChannelPreview/ChannelPreview.tsx

```diff
@@ -110,7 +107,7 @@ export const ChannelPreview = <
       channel.off('message.updated', handleEvent);
       channel.off('message.deleted', handleEvent);
     };
-  }, [channel, isActive, muted]);
+  }, [refreshUnreadCount]);
```

Comment:
```suggestion
  }, [channelUpdateCount]);
```
I think this might be the place you'd want to use it instead?